Makefile.am

```diff
@@ -10,6 +10,7 @@ pkgconfig_DATA = libfastjson.pc
 
 libfastjsonincludedir = $(includedir)/libfastjson
 libfastjsoninclude_HEADERS = \
+	atomic.h \
 	arraylist.h \
```

Comment:
2016-04-08 8:33 GMT+02:00 Rainer Gerhards <notifications@github.com>:

> In Makefile.am
> <https://github.com/rsyslog/libfastjson/pull/50#discussion_r58985046>:
>
> > @@ -10,6 +10,7 @@ pkgconfig_DATA = libfastjson.pc
> >
> >  libfastjsonincludedir = $(includedir)/libfastjson
> >  libfastjsoninclude_HEADERS = \
> > +	atomic.h \
> >  	arraylist.h \
>
> I think we should avoid using it here, I have already worked on the
> ability to not provide json_object_private.h. Especially this header, I
> think, should not be visible to callers - it prevents changing many
> implementation details. I'll investigate further today.
>

+1, that's even better.


-- 
Why is it that all of the instruments seeking intelligent life in the
universe are pointed away from Earth?